src/components/App.js

```diff
@@ -65,6 +66,11 @@ function AppComponent() {
 }
 
 const AppWithProviders = () => {
+  useEffect(()=>{
+    if(Platform.OS === "android") {
+      setTimeout(() => { BackgroundColor.setColor("#271F34") }, 500);
```

Comment:
1. Why you need the timeout?

Reply:
To avoid flickering when app still starting. This would set the color once splash is in the background, could be avoided and tested though.
Here's the repo: https://github.com/ramilushev/react-native-background-color.